assets/js/romo/tooltip.js

```diff
@@ -118,6 +118,28 @@ RomoTooltip.prototype.doSetContent = function(value) {
   this.doPlacePopupElem();
 }
 
+RomoTooltip.prototype.doSetPopupZIndex = function(relativeElem) {
+  var relativeZIndex = Romo.parseZIndex(relativeElem);
+  Romo.setStyle(this.popupElem, 'z-index', relativeZIndex + 1100); // see z-index.css
+}
+
+RomoTooltip.prototype._getPopupMaxAvailableHeight = function(position) {
```

Comment:
@kellyredding - Did you mean to move this method out of the private section?

Reply:
@jcredding no, I didn't great catch - also I didn't even mention these changes in the commit msg - oof!